src/Azure.Functions.Cli/Common/DurableManager.cs

```diff
@@ -0,0 +1,180 @@
+                ColoredConsole.WriteLine($"Input: {status.Input}");
+                ColoredConsole.WriteLine($"Output: {status.Output}");
+                ColoredConsole.WriteLine($"Status: {status.OrchestrationStatus}");
+            }
+        }
+
+        public async Task RaiseEvent(string instanceId, string eventName, object eventData)
+        {
+            var orchestrationInstance = new OrchestrationInstance
+            {
+                InstanceId = instanceId
+            };
+
+            await _client.RaiseEventAsync(orchestrationInstance, eventName, eventData);
+
+            ColoredConsole.WriteLine(Green($"Raised event {eventName} to instance {instanceId} with data {eventData}"));
+        }
+
+        public async Task Rewind(string instanceId, string reason)
+        {
+            var oldStatus = await _client.GetOrchestrationStateAsync(instanceId, false);
+
+            try
+            {
+                await _orchestrationService.RewindTaskOrchestrationAsync(instanceId, reason);
+            }
+            catch (ArgumentOutOfRangeException e)
+            {
+                // DurableTask.AzureStorage throws this error when it cannot find an orchestration instance matching the query
+                throw new CliException("Orchestration instance not rewound. Must have a status of 'Failed', or an EventType of 'TaskFailed' or 'SubOrchestrationInstanceFailed' to be rewound.");
+            }
+            
+            ColoredConsole.WriteLine($"Rewind message sent to instance {instanceId}. Retrieving new status now..");
+
+            // Wait three seconds before retrieving the updated status
+            await Task.Delay(3000);
+            var newStatus = await _client.GetOrchestrationStateAsync(instanceId, false);
+            
+            if (oldStatus != null && oldStatus.Count > 0 
+                && newStatus != null && newStatus.Count > 0)
+            {
+                ColoredConsole.Write(Green("Status before rewind: "));
+                ColoredConsole.Write($"{oldStatus[0].OrchestrationStatus}{Environment.NewLine}");
+                ColoredConsole.Write(Green("Status after rewind: "));
+                ColoredConsole.WriteLine($"{newStatus[0].OrchestrationStatus}");
+            }
+        }
+
+        public async Task StartNew(string functionName, string version, string instanceId, object input)
+        {           
+            if (string.IsNullOrEmpty(instanceId))
+            {
+                instanceId = $"{Guid.NewGuid():N}";
+            }
+
+            await _client.CreateOrchestrationInstanceAsync(functionName, version, instanceId, input);
+
+            var status = await _client.GetOrchestrationStateAsync(instanceId, false);
+
+            if (status != null && status.Count > 0)
+            {
+                ColoredConsole.WriteLine(Green($"Started {status[0].Name} with new instance {status[0].OrchestrationInstance.InstanceId} at {status[0].CreatedTime}."));
```

Comment:
General feedback: string inputs from the user should be surrounded by quotes. This is mainly because the string could contain spaces, and then the full sentence becomes harder to read because you can't tell the input (e.g. the function name or the instance ID) from the sentence itself.

```suggestion
                ColoredConsole.WriteLine(Green($"Started '{status[0].Name}' with new instance '{status[0].OrchestrationInstance.InstanceId}' at {status[0].CreatedTime}."));
```